be/src/services/vocabulary.service.ts

```diff
@@ -26,6 +27,53 @@ export class VocabularyService {
         return this.collection.findOne({_id: new ObjectId(id)}).then(x => <Vocabulary>x);
     }
 
+    public updateVocab(id: string, ifUnmodifiedSince: Date, newVocab: Vocabulary): Promise<Vocabulary> {
+        // @ts-ignore
+        return this.getVocabular(id)
+          .then((dbo: Vocabulary) => {
+              if(!!dbo) {
+                  if(dbo.lastModified.getTime() <= ifUnmodifiedSince.getTime()) {
+                      const filter: Filter<Vocabulary> = {
+                          _id: new ObjectId(id),
+                          lastModified: {
+                              // eslint-disable-rows-line @typescript-eslint/no-unsafe-argument
+                              $lte: ifUnmodifiedSince
```

Comment:
check for `$eq`

Reply:
Will do, I used `lte` because of the `since` in `ifUnmodifiedSince`